### ISSUE TITLE
database credentials security risk

### ISSUE THREAD

**Issue by GoogleCodeExporter** [patel22p/mytinytodo#45]
```
What steps will reproduce the problem?
1. instal mytinytodo with mysql db
2. drop db

What is the expected output? What do you see instead?
expected: worked list
but it shows me error with db password:
>Fatal error: Uncaught exception 'Exception' with message 'Unknown database 
'xsektorxdotorgslashhtodo'' in 
/home/xsektorx/xsektorx.org/htodo/class.db.mysql.php:72 Stack trace: #0 
/home/xsektorx/xsektorx.org/htodo/init.php(26): 
Database_Mysql->connect('localhost', 'DATABASE USER', 'DATABASE PASSWORD', 
'xsektorxdotorgs...') #1 /home/xsektorx/xsektorx.org/htodo/index.php(8): 
require_once('/home/xsektorx/...') #2 {main} thrown in 
/home/xsektorx/xsektorx.org/htodo/class.db.mysql.php on line 72


What version of the product are you using? On what operating system and
browser?
mytinytodo 1.4.2
debian gnu/linux 6.0.2
mysql 5.1.49
php 5.3.3

Please provide any additional information below.
real db user and password was replaced
```

Original issue reported on code.google.com by `xsektorx` on 23 Aug 2011 at 11:03

**Comment by GoogleCodeExporter** [comment]
Original comment by `maxpozd...@gmail.com` on 30 Aug 2011 at 8:36
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
available in v1.5
```

Original comment by `maxpozd...@gmail.com` on 29 Aug 2014 at 11:48
- Changed title: **database credentials security risk**
- Changed state: **Fixed**
